app/presenters/notices/setup/abstraction-alerts/alert-thresholds.presenter.js

```diff
@@ -58,12 +116,14 @@ function _thresholdOptions(licenceMonitoringStations, alertType, alertThresholds
 
   const relevantThresholds = _relevantThresholds(relevantLicenceMonitoringStations)
 
-  return relevantThresholds.map((relevantThreshold) => {
-    const [measureType, thresholdValue, thresholdUnit] = relevantThreshold.split('-')
+  const sortedThresholds = _sortedThresholds(relevantThresholds)
+
+  return sortedThresholds.map((threshold) => {
+    const [measureType, thresholdValue, thresholdUnit] = threshold.split('-')
 
     return {
-      checked: alertThresholds.includes(relevantThreshold),
-      value: relevantThreshold,
+      checked: alertThresholds.includes(threshold),
+      value: threshold,
```

Comment:
this could be better named as thresholdGroup 

i should have named it this initially.